spec/javascripts/Policies/components/HeaderButton.spec.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import React from 'react'
+import { mount } from 'enzyme'
+
+import { HeaderButton } from 'Policies/components/HeaderButton'
+
+it('should render itself', () => {
+  const wrapper = mount(<HeaderButton type='add' onClick={jest.fn()} />)
+  expect(wrapper.find(HeaderButton).exists()).toBe(true)
```

Comment:
I think we were starting to use jest snapshots

Reply:
Only when checking html structure, not just rendering

Reply:
But you're right @damianpm, I'll add some snapshots